sklearn/utils/_weight_vector.pyx.tp

```diff
@@ -139,8 +139,8 @@ cdef class WeightVector{{name_suffix}}(object):
         cdef int idx
         cdef {{c_type}} val
         cdef {{c_type}} mu = 1.0 / num_iter
```

Comment:
Should we use double precision for `mu`, here?

Reply:
I agree here too.